media-sound/pulseaudio-modules-bt/pulseaudio-modules-bt-1.4.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 2019-2021 Gentoo Authors
```

Comment:
I only bumped the latest `1.4-r3 -> 1.4-r4` which has extra restriction on `!media-sound/pulseaudio-daemon[bluetooth]`
Should be no need to bump `1.4` and `1.4-r1` which are just effectively fixed to original pulseaudio version.

Reply:
The main reason I'm pedantic about this is two fold (but related):
1. We want to know everybody is in the same state and avoid confusing dependency conflicts
2. Users often do unexpected things in terms of partial upgrades and so on.

Reply:
Here is a problem, there are multiple `pulseaudio-modules-bt`

Version `1.4` is for `pulseaudio-13.0` and probably will go away when pulseaudio ebuilds of 15.0 are stabliized; for this version the change is only to silence QA warnings.

I opted to remove `1.4-r1` now since this one only differ from revbumped `1.4-r4` in support for `media-sound/pulseaudio-daemon` alternative.